prow/monitoring/BUILD.bazel

```diff
@@ -56,3 +62,30 @@ k8s_objects(
       ":servicemonitor_crd",
    ]
 )
+
+load("//def:configmap.bzl", "k8s_configmap")
+
+k8s_configmap(
+    name = "grafana-datasources",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    namespace = "prow-monitoring",
+    data = {
+        "datasources.yaml": ":datasources.yaml",
+    },
+)
+
+k8s_objects(
+   name = "monitoring-config",
+   objects = [
+      ":grafana-datasources",
+   ]
+)
+
+### workaround before we have k8s_secret at hand
+genrule(
+    name = "alertmanager-prow_secret",
+    srcs = [],
+    outs = ["alertmanager-prow_secret.out"],
+    ### on local shell env, need to set up KUBECONFIG=<path_to_kube_config>
+    cmd = "kubectl create -n prow-monitoring secret generic alertmanager-prow --from-file=alertmanager.yaml=./alertmanager.yaml --from-file=./msg.tmpl --dry-run -o yaml | kubectl apply -f - > $@ 2>&1",
```

Comment:
CI failed: `kubectl` not found
@fejta How should I do this before `k8s_secret` is available.

Reply:
Back to inline for merging and testing.
Will deal with `k8s_secret` with https://github.com/kubernetes/test-infra/pull/13021